<launch>
  <group ns="robot_1">
    <node pkg="tf2_ros" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 0 0 0 base_link camera_color_left" />
    <remap from="left/image_rect" to="/kitti/camera_color_left/image_rect"/>
    <remap from="right/image_rect" to="/kitti/camera_color_right/image_rect"/>
    <remap from="left/camera_info" to="/kitti/camera_color_left/camera_info"/>
    <remap from="right/camera_info" to="/kitti/camera_color_right/camera_info"/>
    <node pkg="rtabmap_ros" type="stereo_odometry" name="stereo_odometry"/>
  </group>
</launch>